code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm

```diff
@@ -31,11 +31,6 @@
 	X.visible_message(span_xenowarning("\The [X] spews forth a wide cone of acid!"), \
 	span_xenowarning("We spew forth a cone of acid!"), null, 5)
 
-	X.add_movespeed_modifier(type, TRUE, 0, NONE, TRUE, 2)
-	start_acid_spray_cone(target, X.xeno_caste.acid_spray_range)
```

Comment:
Wait, dont actually delete the cone creation method call, and the cooldown call in the next line. The cone and following button cooldown wont start without them.

Also keep your changelog and PR description up to date. You're removing the slowdown following the acid cone usage.

Reply:
The slowdown is meant to go, this was discussed in the balance disc

Reply:
Il put the create back in a min